### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by pietrushnic**
This is a proposal for a new, primarily simplified version of the landing page for 3mdeb.com, for it would not be used as a place to land those who will click the 3mdeb logo on the OST2 sponsor page.

zarhus_as_os

### CODE REVIEW

File: content/ost2.md
Comment (artur-rs) [inline content/ost2.md:1]: @pietrushnic change the filename to represent its role, something like `landing_page.md`? There are 3 products visible.

File: assets/images/zavishaos_logo.png
[the remark this answers is not in the document]
Comment (pietrushnic) [reply assets/images/zavishaos_logo.png:1]: Fixed: e0ef71341264

File: content/ost2.md
Comment (pietrushnic) [reply content/ost2.md:1]: Fixed: e0ef71341264

**Comment by artur-rs** [comment]
@pietrushnic I thought Zarhus "Read more" content was a blocker, but I see the target page set to OS info. Approved

**Review by artur-rs — APPROVED** [review]

**Comment by pietrushnic** [comment]
@artur-rs, we will change that link to zarhus.com when the website is ready.

**Pull request merged**